website/docs/d/outputs.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_outputs"
+sidebar_current: "docs-datasource-tfe-state-outputs"
+description: |-
+  Get output values from another organization/workspace.
+---
+# Data Source: tfe_outputs
+
+This data source is used to retrieve the state outputs for a given workspace.
+It enables output values in one Terraform configuration to be used in another.
+
+Note that any output value marked as sensitive from the remote workspace configuration is not redacted in the _current_ configuration when using this datasource. It is the responsibility of the configuration using this datasource to redetermine sensitive values (e.g. using [sensitive()](https://www.terraform.io/docs/language/functions/sensitive.html) or redeclaring a sensitive output value).
```

Comment:
I mentioned this in Slack last week but easy to get lost:

I think the best answer to the sensitive values question we were dealing with is indeed to not touch changing sensitivity from the provider (which this PR does already), with one additional bit: Given that the remote workspace we're pulling from could potentially have sensitive values, the entire value should be marked as sensitive.

We sadly can't do per-output-value marking of sensitivity or any such business, as you can't do the conditional part across the plugin bridge 😭 But Paddy agreed we should probably opt to a safer default and just mark it statically sensitive at the schema level, then, and users can still use the values as normal and even output them by redefining outputs and/or nonsensitive().

TLDR: Given functionality constraints, let's statically mark the whole map as sensitive always.